src/book.py

```diff
@@ -86,7 +86,23 @@ def _read_binance(self, file_path: Path) -> None:
             # Skip header.
             next(reader)
 
-            for _utc_time, account, operation, coin, _change, remark in reader:
+            for rowlist in reader:
+                if version == 1:
+                    _utc_time, account, operation, coin, _change, remark = rowlist
+                if version == 2:
+                    (
+                        user_ID,
+                        _utc_time,
+                        account,
+                        operation,
+                        coin,
+                        _change,
+                        remark,
+                    ) = rowlist
+                else:
+                    log.warning("File version not Supported " + str(file_path))
```

Comment:
This will spam your Log messages. Each line will write a warning.
Just raise Not implemented error

Reply:
this should only raise one per file due to the break afterwords